### PR TITLE
OPENEUROPA-2340: Implement site specific footer 2.x.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by imanoleguskiza — DISMISSED** [review]

**Review by dxvargas — APPROVED** [review]

**Review by dxvargas — DISMISSED** [review]

File: oe_theme.theme [hunk 1]
```diff
@@ -1130,3 +1169,20 @@ function oe_theme_preprocess_pattern_link__preview(array &$variables): void {
   // Pattern preview will pass along a \Drupal\Core\Render\Markup object.
   $variables['url'] = Url::fromUserInput((string) $variables['url']);
 }
+
+/**
+ * Implements template_preprocess_oe_corporate_blocks_footer().
+ *
+ * Adapting data for ECL 1 custom footer.
+ */
+function oe_theme_preprocess_oe_corporate_blocks_footer(&$variables) {
```
Comment (ademarco) [inline oe_theme.theme]: We can remove this on branch 2.x of the theme.

**Review by ademarco — CHANGES_REQUESTED** [review]
I get a warning when rendering the footer:

```
Notice: Undefined index: custom_footer in oe_theme_preprocess_oe_corporate_blocks_ec_footer() (line 264 of /var/www/html/oe_theme.theme). 
```

We should also style the site specific links on the EU footer and test its rendering.

File: oe_theme.theme [hunk 2]
```diff
@@ -262,6 +262,59 @@ function oe_theme_preprocess_oe_corporate_blocks_ec_footer(array &$variables): v
...
+  // Site specific footer.
+  $site_specific_footer = $variables['site_specific_footer'];
+  if (empty($site_specific_footer)) {
+    return [];
+  }
+  if (!empty($site_specific_footer['site_identity'])) {
+    $identity['title'] = $site_specific_footer['site_identity'];
+  }
+  if (!empty($site_specific_footer['social_links'])) {
+    $identity['follow'] = ['label' => t('Follow us:')];
```
Comment (ademarco) [inline oe_theme.theme:289]: We need to keep track of strings that need translations here. Do we already have a ticket for this? Probably we do right?

**Review by ademarco — DISMISSED** [review]

File: oe_theme.theme (hunk 2 above)
Comment (sergepavle) [reply oe_theme.theme:289]: Thank you for highlighting this string. I've added this in OPENEUROPA-2427 ticket (updated table with strings for translation).

**Review by upchuk — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**